batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/utils/DbConnection.java

```diff
@@ -0,0 +1,21 @@
+package com.lftechnology.batch7crud.utils;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+/**
+ * @Author binodnme
+ * Created on 1/18/16
+ */
+public class DbConnection {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)